### PR TITLE
feat(arcgis): convert link to rest format

### PR DESCRIPTION
**Pull request by mspivak-actionengine**

### CODE REVIEW

File: src/components/layers-panel/layers-panel.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import type {
 
 import { useState } from "react";
 import styled, { css } from "styled-components";
-import { load } from "@loaders.gl/core";
+import { forEach, load } from "@loaders.gl/core";
```
Comment (belom88) [inline src/components/layers-panel/layers-panel.tsx:8]: Warning

File: src/components/layers-panel/layers-panel.tsx [hunk 2]
```diff
@@ -197,11 +197,33 @@ export const LayersPanel = ({
...
+    }
+    return urlRest;
+  };
+
   const handleInsertLayer = (layer: {
     name: string;
     url: string;
     token?: string;
   }) => {
+    layer.url = convertUrlToRestFormat(layer.url);
```
Comment (belom88) [inline src/components/layers-panel/layers-panel.tsx:225]: Does it really work for layers? We were about doing it for scenes

File: src/components/layers-panel/layers-panel.tsx [hunk 3]
```diff
@@ -197,11 +197,33 @@ export const LayersPanel = ({
 
   useClickOutside([warningNode], () => setShowExistedError(false));
 
+  const convertUrlToRestFormat = (url: string): string => {
```
Comment (belom88) [inline src/components/layers-panel/layers-panel.tsx:200]: Move to `src\utils\url-utils.ts`. Add a test

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/components/layers-panel/layers-panel.tsx (hunk 2 above)
Comment (mspivak-actionengine) [reply src/components/layers-panel/layers-panel.tsx]: Removed.

File: src/components/layers-panel/layers-panel.tsx (hunk 3 above)
Comment (mspivak-actionengine) [reply src/components/layers-panel/layers-panel.tsx]: Done.

**Review by belom88 — APPROVED** [review]

**Pull request merged**